### PR TITLE
More Java 11 support

### PR DESCRIPTION
**Pull request by jjlauer**
Guava's classpath finder failed on Java 11.  Leveraging ebeans addPackage functionality which supports Java 8 and 11 and is much faster in their latest framework.

### CODE REVIEW

File: ninja-ebean-module/src/main/java/ninja/ebean/NinjaEbeanServerLifecycle.java [hunk 1]
```diff
@@ -168,17 +167,7 @@ public final void startServer(){
...
-                logger.info("Searched and found " + packageClasses.size() + " classes in package " + packageName);
-                for (String packageClass : packageClasses) {
-                    try {
-                        entityClasses.add(Class.forName(packageClass));
-                    } catch (ClassNotFoundException e) {
-                        // should be impossible since Reflections just found 'em
-                        throw new RuntimeException("Something fishy happenend. Unable to find class " + packageClass);
-                    }
-                }
+                serverConfig.addPackage(packageName);
```
Comment (raphaelbauer) [inline ninja-ebean-module/src/main/java/ninja/ebean/NinjaEbeanServerLifecycle.java:170]: Looks simpler... ;)

**Comment by raphaelbauer** [comment]
Nice!

**Comment by jjlauer** [comment]
Thx. Can you release by chance?  Or I'm happy to handle this if you wanna give my Sonatype account access to org.ninjaframework package.

**Comment by rbygrave** [comment]
Hi.  

Just a note that when people get to Ebean 12.3.x  **_IF_** they are using _query beans_ then the query bean generation also generates code to register the entity classes.  For those people there should be no need for any classpath scanning for entity beans etc (as the generated code registers the entity bean classes).

Cheers, Rob.

**Comment by jjlauer** [comment]
hey @rbygrave -- First let me say how much I appreciate how much you put into ebean :-) its my ORM of choice.  Second, I don't use query bean generation myself -- I'm assuming there aren't any issues if you double register them by accident?

**Comment by rbygrave** [comment]
>  I'm assuming there aren't any issues if you double register them by accident?

No problem there.  We can 'double' register if we like - more accurately the generated code to register entity beans is only used if we haven't already registered the entity beans explicitly ourselves.

**Pull request merged**